tools/distrib/python/grpc_prefixed/generate.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2020 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generates grpc-prefixed packages using template renderer.
+
+To use this script, please use 3.7+ interpreter. This script is work-directory
```

Comment:
Nit: You could pretty easily turn this warning comment into a programmatic check.

Reply:
`dataclasses` is 3.7+ only. For Python lower than 3.7, we will got a `ModuleNotFoundError: No module named 'dataclasses'`.

Reply:
Since this script should only be used once, usability of the script is less of a concern. :+1: